### ISSUE TITLE
Is it possible to define the cell height and widht in heatmaply

### ISSUE THREAD

**Issue by lucanico** [talgalili/heatmaply#267]
not sure If this feature is  available. I would like to know if it is currently possible to define cell width an height in  heatmaply().

Apologies in case I missed something.

Thanks

**Comment by Alanocallaghan** [comment]
Can you be a bit more specific? You mean to define how big each cell of the heatmap is in pixels? This is unlikely to be doable since the heatmap sizing includes dendrograms, labels, etc, though I might be able to give you a recipe to try to figure it out in some limited cases.

**Comment by lucanico** [comment]
Thanks for your reply and sorry for not being  clear enough.
Yes,  I’d like to define the cell size to ideally have squared cells . It would be great to have this option, but I understand it  might not be possible without affecting the remaining objects visualised.
Happy to give a try to whatever option is currently available (even if this might only work in specific cases).
Thanks for your help, much appreciated

**Comment by Alanocallaghan** [comment]
Yeah unfortunately this would be really hard to do. The easiest way to do it would be something like

```r
library(heatmaply)
heatmaply(mtcars, show_dendrogram = FALSE, hide_colorbar = TRUE, showticklabels = FALSE)
```

And then save with the same width and height, but even that is not going to be exact.

**Comment by lucanico** [comment]
No worries then ! thank you for your reply
